core/src/main/java/org/gramlab/core/gramlab/frames/ChangePerspective.java

```diff
@@ -0,0 +1,150 @@
+package org.gramlab.core.gramlab.frames;
+
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.GridLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.BorderFactory;
+import javax.swing.ButtonGroup;
+import javax.swing.JButton;
+import javax.swing.JDialog;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JRadioButton;
+import javax.swing.WindowConstants;
+
+import org.gramlab.core.Main;
+import org.gramlab.core.gramlab.util.KeyUtil;
+import org.gramlab.core.umlv.unitex.common.project.manager.GlobalProjectManager;
+import org.gramlab.core.umlv.unitex.frames.FrameUtil;
+
+/**
+ * Dialog to switch between the two perspectives of the Integrated IDE namely, Classic and Project-oriented.
+ * This same dialog would be used both at the start of the IDE, to choose a perspective and switch to other when working with one.
+ * 
+ * @author Mukarram Tailor
+ *
+ */
+
+
+@SuppressWarnings("serial")
+public class ChangePerspective extends JDialog {
+	
+	/**
+	 * 
+	 * @param currPerspective
+	 * the currently active perspective, 'Classic', 'Project-oriented' or 'none'(in case of start up)
+	 * 
+	 * @param args
+	 * command line arguments at start-up
+	 * 
+	 */
+	public ChangePerspective(String currPerspective, String[] args) {
+		super(Main.getMainFrame(), "Choose perspective", true);
+		setContentPane(constructPanel(currPerspective, args));
+		setAlwaysOnTop(true);
+		pack();
+		setLocationRelativeTo(null);
+		setDefaultCloseOperation(WindowConstants.DISPOSE_ON_CLOSE);
+		setVisible(true);
+	}
+
+	private JPanel constructPanel(final String currPerspective, final String[] args) {
+		JPanel p=new JPanel(new BorderLayout());
+		p.setBorder(BorderFactory.createEmptyBorder(5,5,5,5));
+		p.setPreferredSize(new Dimension(400, 200));
+		p.add(new JLabel("Select the perspective you want to use "),BorderLayout.NORTH);
+		JPanel center=new JPanel(new GridLayout(7,7));
+		center.add(new JLabel(""));
+		final JRadioButton classic=new JRadioButton("Classic",true);
+		final JRadioButton projectOriented=new JRadioButton("Project-Oriented",false);
+		ButtonGroup bg=new ButtonGroup();
+		center.add(classic);
+		center.add(projectOriented);
+		bg.add(classic);
+		bg.add(projectOriented);
+		final JLabel error = new JLabel("You are already using "+currPerspective+" Perspective.");
```

Comment:
It would be better to avoid this kind of feedback. What about to disable by default the radio button corresponding to the current active perspective?